packages/core/src/lib/tooltip/tooltip-styles.ts

```diff
@@ -7,6 +7,7 @@ import {
   type FloatingPlacement,
 } from '$lib/floating';
 import { uniqueId } from '$lib/unique-id';
+import { noop } from 'lodash';
```

Comment:
Since we don't use `lodash-es`:

```suggestion
import noop from 'lodash/noop';
```